vertx-config-toml/src/main/java/io/vertx/config/toml/TomlProcessor.java

```diff
@@ -0,0 +1,84 @@
+package io.vertx.config.toml;
+
+import com.moandjiezana.toml.Toml;
+import io.vertx.config.spi.ConfigProcessor;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonObject;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.ByteArrayInputStream;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * <p>
+ * A processor for the <a href='https://github.com/toml-lang/toml'>TOML</a>
+ * format.
+ * </p>
+ * <p>
+ * Currently using the
+ * <a href='https://github.com/mwanji/toml4j'>toml4j library</a>. If a newer
+ * version of this library becomes available, you can override the dependency
+ * (assuming the API does not change).
+ * </p>
+ *
+ * @author <a href='lyndon.codes'>Lyndon Armitage</a>
+ */
+public final class TomlProcessor implements ConfigProcessor {
+
+  public TomlProcessor() {
```

Comment:
Why do we need this empty constructor?

Reply:
I actually added this to be explicit in the fact that nothing was done in the constructor. I can remove it and we can rely on the default constructor created implicitly.